assets/css/dashboard/new-pa-message/new-pa-message.scss

```diff
@@ -14,14 +15,14 @@
   }
 
   .cancel-button {
+    color: $button-primary;
     background-color: transparent;
-    color: $text-link-primary;
+    margin-right: 16px;
     border: none;
-  }
 
-  .submit-button {
-    background-color: $button-primary;
-    color: $button-secondary;
+    &:hover {
+      background-color: #c1e4ff14;
```

Comment:
**question:** i see there are several in-line color values throughout the SCSS changes. do these colors have names we could use instead?

Reply:
Some colors do not have names, but I can create variables based on the button class defined by Mary.